integration-test/src/test/java/com/datastrato/gravitino/integration/test/catalog/jdbc/postgresql/CatalogPostgreSqlIT.java

```diff
@@ -195,6 +197,80 @@ private ColumnDTO[] createColumns() {
     return new ColumnDTO[] {col1, col2, col3};
   }
 
+  private ColumnDTO[] columnsWithSpecialNames() {
+    return new ColumnDTO[] {
+      new ColumnDTO.Builder()
+          .withName("integer")
+          .withDataType(Types.IntegerType.get())
+          .withComment("integer")
+          .build(),
+      new ColumnDTO.Builder()
+          .withName("long")
+          .withDataType(Types.LongType.get())
+          .withComment("long")
+          .build(),
+      new ColumnDTO.Builder()
+          .withName("float")
+          .withDataType(Types.FloatType.get())
+          .withComment("float")
+          .build(),
+      new ColumnDTO.Builder()
+          .withName("double")
+          .withDataType(Types.DoubleType.get())
+          .withComment("double")
+          .build(),
+      new ColumnDTO.Builder()
+          .withName("decimal")
+          .withDataType(Types.DecimalType.of(10, 3))
+          .withComment("decimal")
+          .build(),
+      new ColumnDTO.Builder()
+          .withName("date")
+          .withDataType(Types.DateType.get())
+          .withComment("date")
+          .build(),
+      new ColumnDTO.Builder()
+          .withName("time")
+          .withDataType(Types.TimeType.get())
+          .withComment("time")
+          .build(),
+      new ColumnDTO.Builder()
+          .withName("binary")
+          .withDataType(Types.TimestampType.withoutTimeZone())
+          .withComment("binary")
+          .build()
+    };
+  }
+
+  @Test
+  void testCreateTableWithSpecialColumnNames() {
+    // Create table from Gravitino API
+    ColumnDTO[] columns = columnsWithSpecialNames();
+
+    NameIdentifier tableIdentifier =
+        NameIdentifier.of(metalakeName, catalogName, schemaName, tableName);
+    Distribution distribution = Distributions.NONE;
+
+    SortOrder[] sortOrders = new SortOrder[0];
+    Partitioning[] partitioning = Partitioning.EMPTY_PARTITIONING;
```

Comment:
I think you should use `Transform[0]` instead of `Partitioning`. `Partitioning` should not be exposed to the user.

Reply:
Addressed.

Reply:
I have replaced `new Transform[0]` with `Transforms.EMPTY_TRANSFORM`.